__docs__/wonder-blocks-accordion/accordion.stories.tsx

```diff
@@ -282,9 +465,60 @@ SingleSection.parameters = {
     },
 };
 
+/**
+ * This is an example of an Accordion with many sections, as well as
+ * a lot of content within each section.
+ */
+export const LongSections: StoryComponentType = {
```

Comment:
**suggestion:** Disable this chromatic snapshot as we cannot visually test this in chromatic.